tests/test_axis_internal.py

```diff
@@ -26,7 +26,7 @@ def test_axis_regular_uoflow(axtype, function):
     assert 10 == ax.index(1.01)
     assert 10 == ax.index(23)
 
-    assert 10 == ax.size
+    assert 10 == len(ax)
```

Comment:
`ax.size` is also available and 10, correct?

Reply:
Both should be tested, right. Anyway, the tests here are largely overlapping with test_axis_public.py, I will merge the two files of tests next.

Reply:
Good! Tests need cleanup.